artcommon/artcommonlib/konflux/konflux_build_record.py

```diff
@@ -85,6 +85,7 @@ def __init__(
         pipeline_commit: str = '',
         schema_level: int = 0,
         ingestion_time: datetime = None,
+        component: str = '',
```

Comment:
I think we should have a field called "konflux_component" in all 3 build records. 2 reasons
- `component` is an overloaded term, so this makes it clear
- the field name is standardized across tables, to keep it similar to fields like name, version, release which each build has.